content/en/docs/concepts/scheduling-eviction/scheduling-framework.md

```diff
@@ -222,6 +222,26 @@ This is an informational extension point. Post-bind plugins are called after a
 Pod is successfully bound. This is the end of a binding cycle, and can be used
 to clean up associated resources.
 
+### EnqueueExtension
```

Comment:
This should go after PreEnqueue.

I wonder if we should rename `## Extension points` to `## Interfaces`, so that it's not confused with https://kubernetes.io/docs/reference/scheduling/config/#extension-points, which actually form part of the API.

Reply:
Sounds good, I rename this along with some wording changes in the doc.